internal/server/websocketServer.go

```diff
@@ -0,0 +1,191 @@
+package server
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/dicedb/dice/config"
+	"github.com/dicedb/dice/internal/clientio"
+	diceerrors "github.com/dicedb/dice/internal/errors"
+	"github.com/dicedb/dice/internal/ops"
+	"github.com/dicedb/dice/internal/querywatcher"
+	"github.com/dicedb/dice/internal/server/utils"
+	"github.com/dicedb/dice/internal/shard"
+	dstore "github.com/dicedb/dice/internal/store"
+	"github.com/gorilla/websocket"
+)
+
+const Qwatch = "QWATCH"
+const Qunwatch = "QUNWATCH"
+const Subscribe = "SUBSCRIBE"
+
+var unimplementedCommandsWebsocket map[string]bool = map[string]bool{
+	Qwatch:    true,
+	Qunwatch:  true,
+	Subscribe: true,
+}
+
+type WebsocketServer struct {
+	querywatcher    *querywatcher.QueryManager
```

Comment:
Is `watchChan`, `querywatcher` required for `WebsocketServer`?

Reply:
Done.